apps/design-system-docs/docs/components/packages/wp-components/data.ts

```diff
@@ -234,7 +234,7 @@ export const data: ComponentData[] = [
 		status: 'use-with-caution',
 		docs: 'https://wordpress.github.io/gutenberg/?path=/docs/components-datetimepicker--docs',
 		notes:
-			'If possible, use `DateCalendar` from `@automattic/components` instead. For the input fields, consider using an `InputControl` with `type="datetime-local"`.',
+			'If possible, use `DateCalendar` from `@automattic/components` instead. For the input fields, consider using an `TextControl` with `type="date"` or `type="datetime-local"`.',
```

Comment:
Nite: should this info be updated to `@automattic/ui` ?

Reply:
This was done in #104328 (hello, time traveler 😄).

Reply:
You haven't seen anything yet. I've got 200+ notifications left to travel to :lolsob: